src/lib/storage/value_segment.hpp

```diff
@@ -45,7 +45,9 @@ class ValueSegment : public AbstractSegment {
   size_t estimate_memory_usage() const final;
 
  protected:
-  // Implementation goes here
+  std::vector<T> _value_vector;
+  std::vector<bool> _is_null_vector;
```

Comment:
Naming these after their container types seems redundant, is there a reason for using [hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation)?

Reply:
If you mean the leading underscores: [Private / protected members / methods: like variables / functions with leading underscore, e.g., _on_execute().](https://github.com/hyrise/hyrise/blob/master/CONTRIBUTING.md#formatting-and-naming)
If you mean the added container types - we should probably change that